shared/src/androidTest/kotlin/com/kcc/kmmhackathon/shared/network/MealEndpointTest.kt

```diff
@@ -0,0 +1,12 @@
+package com.kcc.kmmhackathon.shared.network
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class MealEndpointTestAND {
```

Comment:
Have to differentiate the class name (hence the "AND" on the end) but having the filename be the same across CommonTest, AndroidTest, IOSTest isn't an issue